packages/fx-core/tests/plugins/solution/solution.getQuestions.test.ts

```diff
@@ -625,4 +625,178 @@ describe("getQuestionsForScaffolding()", async () => {
       }
     }
   });
+  it("getQuestionsForUserTask - addFeature: cannot add message extension for notification bot", async () => {
+    sandbox.stub(featureFlags, "isPreviewFeaturesEnabled").returns(true);
```

Comment:
Since some feature flag has been removed since GA, does any flag can be removed?